collector/collector_manager.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/google/cadvisor/info/v1"
 )
 
+const metricLabelPrefix = "io.cadvisor.metric."
```

Comment:
An identifying name  -eg . io.cadvisor.metric.nginx. This allows for multiple config to be specified.
If there are any processing errors, the name is what users will see in their error logs - "failed to parse config nginx"